vignettes/fims-path-maturity.Rmd

```diff
@@ -0,0 +1,549 @@
+---
+title: "FIMS Path: Maturity"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{FIMS Path: Maturity}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+This vignette describes the hierarchical structure of FIMS by describing the linkages, or *path* through the FIMS model using maturity as an example. The vignette walks through how input from R is carried into the core FIMS calculations. 
```

Comment:
Is this vignette intended for developers only? Or would we imagine future users who are not also developers would need to understand this as well? Perhaps a sentence highlighting who the audience is would be helpful.

Reply:
I added: _The vignette is tailored to FIMS developers or others interested in understanding the FIMS core architecture._